app.rb

```diff
@@ -90,7 +90,6 @@ def create_a_student
     student = Student.new(classroom: @classroom, age: age, name: name, parent_permission: has_permission)
     @people.push(student)
     puts 'Student created successfully'
-    puts
     choose_a_number
   end
```

Comment:
- [ ] in the `create_a_student` method you are printing to the console getting information from the user and saving the information this is not in line with the single responsibility rule, have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more information.